protostar/commands/test/testing_summary.py

```diff
@@ -155,3 +173,42 @@ def _get_preprocessed_core_testing_summary(
             test_suites_result.append(f"{total_count} total")
 
         return test_suites_result
+
+    def _get_slowest_list(
+        self,
+        failed_and_passed_list: List[Union[PassedTestCase, FailedTestCase]],
+        count: int,
+    ) -> List[Union[PassedTestCase, FailedTestCase]]:
+        lst = sorted(
+            failed_and_passed_list, key=lambda x: x.execution_time, reverse=True
+        )
+        return lst[: min(count, len(lst))]
+
+    def _get_formatted_slow_tests(self, count: int) -> str:
```

Comment:
The name suggests returning a list of strings and that this function is a getter.

```suggestion
    def _format_slow_test_cases_list(self, count: int) -> str:
```